libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1683,7 +1686,10 @@ string YulUtilFunctions::copyByteArrayToStorageFunction(ArrayType const& _fromTy
 		templ("panic", panicFunction());
 		templ("byteArrayLength", extractByteArrayLengthFunction());
 		templ("dstDataLocation", arrayDataAreaFunction(_toType));
+		if (fromStorage)
+			templ("srcDataLocation", arrayDataAreaFunction(_fromType));
 		templ("clearStorageRange", clearStorageRangeFunction(*_toType.baseType()));
+		templ("srcStride", to_string(fromStorage ? 1 : 0x20));
```

Comment:
And this, too.

Reply:
Not sure if we should call it `stride`, as stride is in bytes (bytes per one element), and for storage types we use `storageSize` when dealing with storage size. So maybe calling it `srcSize` here?

Reply:
I'll call it `increment`